tower-batch/src/service.rs

```diff
@@ -57,8 +57,8 @@ where
         T::Error: Send + Sync,
         Request: Send + 'static,
     {
-        // XXX(hdevalence): is this bound good
-        let bound = 1;
+        // We use the max_items as maximum number of requests.
```

Comment:
Do we need to add to this comment more info like why do we think is a good idea to have the bound the same as the max_items ?